src/app/components/tile/index.html

```diff
@@ -260,7 +262,15 @@
       <div class="screenshot screenshot-danger"><img src="~/assets/img/guidelines/tile/bad-example-3.png" style="width: 100%;" /></div>
     </sky-column>
   </sky-row>
+  </div>
 
+  <div class="sky-subsection-heading">Options</div>
+    <div class="sky-emphasized">Help</div>
+      <p class="sky-text-block">
+        For tasks where users require documentation for a tile, they invoke help from the tile header. If only a small number of items in the tile need explanation, use <a routerLink="/design/guidelines/user-assistance" fragment="inline-help">element-level help</a> on those items instead. In general you should not combine tile-level help and element-level help on items within the tile.
```

Comment:
In the second sentence, I'd change "a small number of" to "a few" to tighten up and make this flow more naturally. ... I'd delete "on those items" to tighten up because I don't think that changes the meaning here. ... In the last sentence, add a comma after "general." ... I'd also change "you should not" to "don't" to be more firm that this is a guideline, not a suggestion. ("In general" already conveys that this isn't a hard and fast rule.)

Reply:
updated